core/src/test/java/org/elasticsearch/search/aggregations/pipeline/BucketScriptIT.java

```diff
@@ -238,6 +243,52 @@ public void testInlineScript2() {
         }
     }
 
+    public void testInlineScript3() {
+        SearchResponse response = client()
+            .prepareSearch("idx")
+            .addAggregation(
+                dateRange("range")
+                    .field(FIELD_5_NAME)
+                    .addRange("now-1d", "now")
```

Comment:
When you run tests, a seed is automatically chosen and returned in case of errors. This way, you can re-run the test with this particular seed so that these random vars get exactly the same values. This would not be the case if `now` is used.